spyder_kernels/customize/namespace_manager.py

```diff
@@ -12,10 +12,22 @@
 from spyder_kernels.py3compat import PY2
 
 
-def _get_globals():
-    """Return current namespace"""
-    ipython_shell = get_ipython()
-    return ipython_shell.user_ns
+def _get_globals_locals():
```

Comment:
We already have a method called `_get_current_namespace` in SpyderKernel, which is almost exactly as this one. I think it'd be better to use that instead.

Reply:
I want to get both global and local separately as they are different arguments to `exec`. `_get_current_namespace` returns a mix of the two in a single dictionnary.